README.md

```diff
@@ -0,0 +1,18 @@
+Workflow CA - Noroff
```

Comment:
This looks incomplete. You can use this template https://github.com/rhappdev/nodejs-template/blob/master/Readme.md. Also, You need to mention e2e, prettier, eslint, pre-commit hook etc